.github/workflows/test.yml

```diff
@@ -0,0 +1,38 @@
+name: Run tests
+
+on:
+  workflow_run:
+    workflows: [build]
+    types: [completed]
```

Comment:
This is actually a security risk as the `workflow_run` trigger starts a run with elevated permissions, which means that running any kind of untrusted code (e.g. from an outside PR). 

The [default permissions ](https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token)on the GITHUB_TOKEN are rw on almost everything, so even with no secrets exposed in the env a lot of damage could be done.